README.md

```diff
@@ -342,11 +347,9 @@ Ensure you're running Podman 3.1.0 or higher because of bugs in older versions.
 config {
   logging = {
     driver = "journald"
-    options = [
```

Comment:
Was this really supposed to be a list?

Reply:
Hum...good point, it seems like it's supposed to be map 👍